pom.xml

```diff
@@ -452,7 +452,7 @@
     <version.org.jboss.arquillian.container.weld>1.0.0.CR8</version.org.jboss.arquillian.container.weld>
     <version.org.jboss.arquillian.container.tomcat>1.0.0.CR7</version.org.jboss.arquillian.container.tomcat>
     <version.org.jboss.shrinkwrap.resolver>2.2.2</version.org.jboss.shrinkwrap.resolver>
-    <version.org.jboss.jboss-transaction-spi>7.5.2.Final</version.org.jboss.jboss-transaction-spi>
+    <version.org.jboss.jboss-transaction-spi>7.5.3.Final-SNAPSHOT</version.org.jboss.jboss-transaction-spi>
```

Comment:
I updated to 7.6.0.Final-SNAPSHOT in jboss-transaction-spi but you need to do a SPI release now and then change this to the right version number before release. Let @mmusgrov or myself know if you need help to do the SPI release:  https://developer.jboss.org/wiki/NarayanaReleaseProcess